Formula/get_iplayer.rb

```diff
@@ -119,7 +119,9 @@ def install
 
     inreplace ["get_iplayer", "get_iplayer.cgi"] do |s|
       s.gsub!(/^(my \$version_text);/i, "\\1 = \"#{pkg_version}-homebrew\";")
-      s.gsub! "#!/usr/bin/env perl", "#!/usr/bin/perl"
+      on_macos do
+        s.gsub! "#!/usr/bin/env perl", "#!/usr/bin/perl"
+      end
```

Comment:
I think we can remove this entirely since keg relocation takes care of this now.